src/app/article-list/article-list.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+    <div class="row articles-list-section">
+        <div class="col-lg-3">
+            <div class="article-image">
+                <img class="img-responsive" src="../../assets/image1.jpg" alt="Article Image">
+            </div>
+        </div>
+        <div class="col-lg-9">
+            <div class="atricle-description">
```

Comment:
I see the word `article` is misspelled in this file a couple of times

Reply:
**Typos Rectified**